chrome/content/exteditor.js

```diff
@@ -229,6 +229,60 @@ function updateEditor() {
     }
 }
 
+function extEditorCompFields2Recipients(msgCompFields) {
+    // Copied and tweaked from addressingWidgetOverlay.js to keep functionalities as of Thunderbird 60
+
+    let listbox = document.getElementById("addressingWidget");
+    let newListBoxNode = listbox.cloneNode(false);
+    let listBoxColsClone = _menulistFriendlyClone(listbox.itemChildren[0]);
+    newListBoxNode.appendChild(listBoxColsClone);
+    let templateNode = _menulistFriendlyClone(listbox.itemChildren[0]);
```

Comment:
`_menulistFriendlyClone` is apparently not intended to be used by add-ons,so it may break at some future date.